dateparser/freshness_date_parser.py

```diff
@@ -147,7 +147,7 @@ def get_kwargs(self, date_string):
 
         kwargs = {}
         for num, unit in m:
-            kwargs[unit + 's'] = int(num)
+            kwargs[unit + 's'] = float(num.replace(",", "."))
```

Comment:
2. Convert unit type to float instead of int, (replacing comma with period for [multi-language support](https://github.com/scrapinghub/dateparser/pull/876#discussion_r571985940))